persistent-sqlite/Database/Persist/Sqlite.hs

```diff
@@ -467,7 +467,7 @@ sqlColumn (Column name isNull typ def _cn _maxLen ref) = T.concat
     , mayDefault def
     , case ref of
         Nothing -> ""
-        Just (table, _) -> " REFERENCES " <> escape table
+        Just (table, _) -> if noRef then "" else " REFERENCES " <> escape table
```

Comment:
So the other option would be to set `ref` on each `Column` to `Nothing`. I kind of like that because it reduces the amount of branching. But I could also see an argument that it's confusing to modify the data of `Column`, which actually hasn't changed, and the `Bool` parameter is better because it shows that the data is being overridden. Thoughts?

Reply:
We are in the function which is being mapped over the columns, so it seemed easiest to do the work here.  Does that sound reasonable?

Reply:
Ok cool. PR LGTM!

Reply:
Thanks @MaxGabriel !